internal/machine/lease.go

```diff
@@ -0,0 +1,52 @@
+package machine
```

Comment:
Is it a problem that we have the machines package split across `internal/command/machine` and `internal/machine`?

Reply:
We can rename the package if it's confusing, but it's not really an issue.  If both packages need to be used at the same time we will just need to define an alias during import.